packages/create-llama/helpers/llama-pack.ts

```diff
@@ -1,29 +1,70 @@
 import fs from "fs/promises";
+import got from "got";
 import path from "path";
-import { LLAMA_HUB_FOLDER_PATH, LLAMA_PACK_CONFIG_PATH } from "./constant";
+import {
+  LLAMA_PACK_FOLDER,
+  LLAMA_PACK_FOLDER_PATH,
+  LLAMA_PACK_OWNER,
+  LLAMA_PACK_REPO,
+} from "./constant";
 import { copy } from "./copy";
 import { templatesDir } from "./dir";
 import { installPythonDependencies } from "./python";
 import { getRepoRawContent } from "./repo";
 import { InstallTemplateArgs } from "./types";
 
+const getLlamaPackFolderSHA = async () => {
+  const url = `https://api.github.com/repos/${LLAMA_PACK_OWNER}/${LLAMA_PACK_REPO}/contents`;
+  const response = await got(url, {
+    responseType: "json",
+  });
+  const data = response.body as any[];
+  const llamaPackFolder = data.find((item) => item.name === LLAMA_PACK_FOLDER);
+  return llamaPackFolder.sha;
+};
+
+const getLLamaPackFolderTree = async (
+  sha: string,
+): Promise<
+  Array<{
+    path: string;
+  }>
+> => {
+  const url = `https://api.github.com/repos/${LLAMA_PACK_OWNER}/${LLAMA_PACK_REPO}/git/trees/${sha}?recursive=1`;
+  const response = await got(url, {
+    responseType: "json",
+  });
+  return (response.body as any).tree;
+};
+
 export async function getAvailableLlamapackOptions(): Promise<
   {
     name: string;
     folderPath: string;
-    example: boolean | undefined;
   }[]
 > {
-  const libraryJsonRaw = await getRepoRawContent(LLAMA_PACK_CONFIG_PATH);
-  const libraryJson = JSON.parse(libraryJsonRaw);
-  const llamapackKeys = Object.keys(libraryJson);
-  return llamapackKeys
-    .map((key) => ({
-      name: key,
-      folderPath: libraryJson[key].id,
-      example: libraryJson[key].example,
-    }))
-    .filter((item) => !!item.example);
+  const EXAMPLE_RELATIVE_PATH = "/examples/example.py";
```

Comment:
so we don't need to read a metadata file anymore? we're just checking if a example.py file exists? isn't that too slow?